logstash-core/lib/logstash/environment.rb

```diff
@@ -40,8 +40,18 @@ module Environment
             Setting::String.new("http.host", "127.0.0.1"),
             Setting::PortRange.new("http.port", 9600..9700),
             Setting::String.new("http.environment", "production"),
+            Setting::String.new("queue.type", "persisted", true, ["persisted", "memory", "synchronous"]),
+            Setting::Numeric.new("queue.page_capacity", 250 * 1024 * 1024),
+            Setting::Numeric.new("queue.queue_capacity", 250 * 1024 * 1024)
```

Comment:
I pushed 70e102e to set the  `queue.queue_capacity` to Numeric 0 since for now this "capacity" is the maximum number of events and not the queue size.